app/models/evaluation.rb

```diff
@@ -67,4 +67,52 @@ def self.filters_to_json
       }
     ].to_json
   end
+
+  def self.to_csv(ids = [])
+
+    csv = ''
+
+    evaluation_columns = Evaluation.new.attributes.keys
+    evaluation_columns << "evaluation_id"
+
+    excluded_attributes = ["created_at", "updated_at", "id", "site_id", "source_id"]
+
+    evaluation_columns.delete_if { |k, v| excluded_attributes.include? k }
+
+    evaluation_columns << "wpda_id"
```

Comment:
You could refactor the following lines by adding those values in an array and perform the `<<` assignment just once

Reply:
Many thanks @Levia 👍